app/apptesting/test_helpers.go

```diff
@@ -0,0 +1,67 @@
+package apptesting
+
+import (
+	"github.com/Stride-Labs/stride/app"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	tmtypes "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+const Bech32Prefix = "stride"
+
+type AppTestHelper struct {
+	suite.Suite
+
+	App         *app.StrideApp
+	Ctx         sdk.Context
+	QueryHelper *baseapp.QueryServiceTestHelper
+	TestAccs    []sdk.AccAddress
+}
+
+func (s *AppTestHelper) Setup() {
+	config := sdk.GetConfig()
+	config.SetBech32PrefixForAccount(Bech32Prefix, Bech32Prefix+sdk.PrefixPublic)
```

Comment:
Awesome job powering through this, really was a confusing issue 💪 
The spreadsheet you made took some real dedication!